core/solver/cgs.hpp

```diff
@@ -46,36 +46,14 @@ namespace gko {
 namespace solver {
 
 
-template <typename>
-class CgsFactory;
-
-
-/**
- * CGS or the conjugate gradient square method is an iterative type Krylov
- * subspace method which is suitable for general systems.
- *
- * The implementation in Ginkgo makes use of the merged kernel to make the best
- * use of data locality. The inner operations in one iteration of CGS are merged
- * into 3 separate steps.
- *
- * @tparam ValueType precision of matrix elements
- */
```

Comment:
Why did you delete the documentation?

Reply:
That's a good question, definitely by accident, I'll add it back.